views/components/BeerCard.jsx

```diff
@@ -0,0 +1,29 @@
+const React = require('react');
+
+
+function BeerCard(props) {
+  return (
+    <div key={props.i}>
+        <img src={props.beerObj.image_url} height="100" alt="" />
+        <h3>{props.beerObj.name}</h3>
+        <h5>{props.beerObj.tagline}</h5>
+        <p>{props.beerObj.description}</p>
+        {
+            props.details && 
+            ( <>
+                <h5>Food Pairing</h5>
+              <ul>
+                { 
+                  props.beerObj.food_pairing.map( (pair, i) => {
+                  return (<li key="i">{pair}</li>);
```

Comment:
In the key, you are actually not referencing the index. It works without it but if you want to put it, you should write it like this:

`return (<li key={i}>{pair}</li>);`